### PR TITLE
get activity from ContextWrapper (fix #7340)

### LINKED ISSUE #7340: Navigation with maps.me not starting
##### Detailed steps causing the problem:
- Have maps.me installed
- Open cache details of a cache
- Select Navigate -> maps.me

##### Actual behavior after performing these steps:
Nothing happens, maps.me not starting

##### Expected behavior after performing these steps:
maps.me should be started with cache coordinates as destination

##### Version of c:geo used:
2019.02.23

##### Is the problem reproducible for you?
Yes (for me) and reported via FB

##### System information:
Tested on my Samsung S7, Android 8.0

### PR DESCRIPTION
**Pull request by bekuno**
fix #7340 (Navigation with maps.me not starting)

### CODE REVIEW

File: main/src/cgeo/geocaching/apps/navi/MapsMeApp.java [hunk 1]
```diff
@@ -72,4 +75,17 @@ public boolean isInstalled() {
         return true;
     }
 
+    private static Activity getActivity(final Context context) {
+        if (context == null) {
```
Comment (Bananeweizen) [inline main/src/cgeo/geocaching/apps/navi/MapsMeApp.java]: The first if is superfluous. instanceof is always false for null, therefore a null value bypasses the instanceof checks and returns null from the end of the method already.

**Comment by Lineflyer** [comment]
Please kindly target this for release branch.

File: main/src/cgeo/geocaching/apps/navi/MapsMeApp.java (hunk 1 above)
Comment (bekuno) [reply main/src/cgeo/geocaching/apps/navi/MapsMeApp.java]: never trust a sample code :-)
thanks for pointing.

**Comment by bekuno** [comment]
> Please kindly target this for release branch.

How to remove now the extra commits?

**Comment by Lineflyer** [comment]
I am sadly not yet a git expert and can only refer to this:
https://github.com/cgeo/cgeo/wiki/How-to-get-a-bug-fix-into-the-release

Maybe it helps.

**Comment by bekuno** [comment]
@Bananeweizen can you push the PR also to release?

**Comment by UniQP** [comment]
@bekuno Should I retarget the PR to release branch (without the extra commits)?

**Comment by bekuno** [comment]
> @bekuno Should I retarget the PR to release branch (without the extra commits)?

This would be fine.
Can you provide me after that a short todo for the next time?

**Comment by UniQP** [comment]
On command line:
`git rebase --onto newBase oldBase`

What I did for this branch:
```
git checkout i#7340_Navigation_with_maps.me_not_starting
git rebase master
git rebase --onto release master
```
The first rebase ensures that master is your old base.

Alternatively you can do
```
git checkout i#7340_Navigation_with_maps.me_not_starting
git rebase --onto release HEAD~2
```
Advantage: You might have less merge conflicts.
Disadvantage: You need to count the number of commits (2 in our case).

**Pull request merged**